README.md

```diff
@@ -98,7 +100,7 @@ The Web Clipper is a browser extension that allows you to save web pages and scr
 - Search functionality.
 - Geo-location support.
 - Supports multiple languages
-- External editor support - open notes in your favorite external editor with one click in Joplin.
+- External editor support - open notes in your favourite external editor with one click in Joplin.
```

Comment:
The reason I originally ran my spell check (using en-US) was because of the inconsistent usage of spelling. After complaints, I switched my spelling engine to en-GB then ran it again to fix all the spelling errors.